src/freenas/usr/local/sbin/ix-update

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+readonly UPDATE_FAILED_LOG="/data/update.failed"
+
+if [ ! -f /data/need-update ]
+then
+    exit 0
+fi
+
+migrate > $UPDATE_FAILED_LOG 2>&1
```

Comment:
I doubt that this is going to fail but still for any reason if it might, I think we are not going to exit the script which means that we remove the error log below anyways. We could perhaps check return code ?

Reply:
This script is `#!/bin/bash -e`, it will exit on first command failure.